src/Admin/Customizer/CustomControls/WP_Customize_Fields_Repeater_Control.php

```diff
@@ -638,6 +665,7 @@ public function template($index = 9999999999999)
                     <?php $this->repeater_textarea_field($index, 'field_options', '', __('Options', 'mailoptin'), __('Enter a comma-separated list of options', 'mailoptin')); ?>
                     <?php $this->repeater_text_field($index, 'hidden_value', '', __('Value', 'mailoptin'), __('Enter the value for this hidden field', 'mailoptin')); ?>
 
+                    <?php $this->repeater_select_field($index, 'list_country_options_field_options', $list_country_name_select_type, '', __('Show Country In', 'mailoptin')); ?>
```

Comment:
Change every instances of `list_country_options_field_options` to `country_field_options`